viewing_party/party.py

```diff
@@ -1,23 +1,144 @@
 # ------------- WAVE 1 --------------------
 
+from cmath import rect
+from time import monotonic
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+
+    if title and genre and rating:
+        movie_dict['title'] = title 
+        movie_dict['genre'] = genre
+        movie_dict['rating'] = rating
+    else:
+        return None
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie):
+    for movie_dicts in user_data['watchlist']:
+        if movie_dicts['title'] == movie:
+            user_data['watched'].append(movie_dicts)
+            user_data['watchlist'].remove(movie_dicts)    

```

Comment:
Style: `movie_dicts` is only a single dict so shouldn't have a plural name:

```suggestion
    for movie_dict in user_data['watchlist']:
        if movie_dict['title'] == movie:
            user_data['watched'].append(movie_dict)
            user_data['watchlist'].remove(movie_dict)

```